parsers/barcode_aztec.go

```diff
@@ -0,0 +1,42 @@
+package parsers
+
+import (
+	"strconv"
+
+	"github.com/ingridhq/zebrash/elements"
+	"github.com/ingridhq/zebrash/printers"
+)
+
+func NewBarcodeAztecParser() *CommandParser {
+	const code = "^BO"
+
+	return &CommandParser{
+		CommandCode: code,
+		Parse: func(command string, printer *printers.VirtualPrinter) (interface{}, error) {
+			barcode := &elements.BarcodeAztec{
+				Orientation: printer.DefaultOrientation,
+			}
+
+			parts := splitCommand(command, code, 0)
+			if len(parts) > 0 && len(parts[0]) > 0 {
+				barcode.Orientation = toFieldOrientation(parts[0][0])
+			}
+
+			if len(parts) > 1 {
+				if v, err := strconv.Atoi(parts[1]); err == nil {
+					barcode.Magnification = v
+				}
+			}
+
+			if len(parts) > 3 {
+				if v, err := strconv.Atoi(parts[3]); err == nil {
```

Comment:
what is under `parts[2]` and why it's skipped here?

Reply:
It is just some option that we do not support
```
eci: Whether or not the bar code data uses ECI (extended channel interpretation) to switch character sets. Valid values are Y and N. The default value is N (no ECI).
```

Reply:
added TODO for handling eci